content/francineblanc/how-to-use-git-for-beginners.md

```diff
@@ -0,0 +1,219 @@
+
+## Making Changes and Adding Files to the Staging Area
+In order for Git to start tracking changes, files need to be added to the Staging Area. This can be done with the command:
+```
+git add <filename>
+```
+where filename is the name and extension of the file to be added to the staging area. Multiple files can also be added with the syntax:
+
+```
+git add <filename1> <filename2> <filename3>
+```
+
+If all files in the Working Directory need to be tracked, the syntax
+
+```
+git add .
+```
+can be used.
+
+Once a file has been added, ```git status``` can be run in the terminal/command prompt to check that the file is in the Staging Area. Git will show the changed files to be committed in <span style="color:green">green</span> text like so:
+
+![git-status-untracked-with-gitignore](git-add-and-status.png)
+
+You can find out more about the additional extensions to ```git add``` in the [Add] section of the Codecademy Docs for Git.
+
+### Oops! I didn't want to add that!
+There may be occasions where a file has been incorrectly added to the Staging Area. This can be undone with the command ```git reset <filename>```. This will not affect the changes done to the file in any way, it will simply remove it from the staging area. 
+
+In the case where all files needs to be unstaged, the command ```git reset``` can be run without adding further arguments.
+
+Note that as of ```git version 2.24``` there is alternative syntax to unstage files, which is ```git restore --staged <filename>``` where ```filename``` is the name of the file to be unstaged.
+
+## Committing Changes
+Committing is generally the last step of the Git workflow and is generally thought of being a snapshot of a project at a particular time. In this stage, changes in the Staging Area are saved inside the local repository, which is achieved with the command ```git commit -m``` followed by a space and a short message explaining the commit in quotes, for example:
+```
+git commit -m "Add title and description to intro file"
+```
+There are general conventions for writing a commit message including making sure they are no more than 50 characters long and writing them in the imperative or present tense. Messages should always be clear and confirm what the change made is so that others can easily see what has changed. 
+
+Committing is an important step in the Git workflow, as once a change has been committed, it can be recalled at a later date or the project can be rewound to that particular version. Commit history can be viewed with ```git log```, which will display a list of commits in chronological order (the most recent commits being at the top), along with information such as the author of the commit, the date and time of the commit and the commit message.
+
+### Oops! I didn't want to commit that!
+In the situation where the last commit needs to be undone, the ```git reset``` command can again be used, wtih some modifications. 
```

Comment:
```suggestion

In the situation where the last commit needs to be undone, the `git reset` command can again be used, with some modifications. 
```

Reply:
💁‍♀️ Something I like to do to quickly check for typos/grammer, is to pop my entire article into Grammarly, and make edits as needed to the file through my text editor or IDE

Reply:
I have run the article through Grammarly and have updated the spelling/grammar as suggested by the program